presto-spi/src/test/java/com/facebook/presto/spi/page/TestPageCodecMarker.java

```diff
@@ -84,4 +87,20 @@ public void testSummaryString()
             assertTrue(allMarkersSummary.contains(marker.name()));
         }
     }
+
+    @Test
+    public void testComputeCRC() {
+        Slice data = utf8Slice("This is a random text");
+        byte pageCodecmarker = 4;
+        int positionCount = 12444567;
+        int uncompressedSize = 10020078;
+        long crc = computeSerializedPageChecksum(data, pageCodecmarker, positionCount, uncompressedSize);
+        assertEquals(1051058195, crc);
+
+        pageCodecmarker = 0;
```

Comment:
this is a strange case; if pageCodecmarker is 0, we won't be calculating the checksum

Reply:
Using 7 where all bits are set.